pkg/controller/autoimport/autoimport_controller.go

```diff
@@ -79,6 +81,16 @@ func (r *ReconcileAutoImport) Reconcile(ctx context.Context, request reconcile.R
 		return reconcile.Result{}, err
 	}
 
+	// ensure the klusterlet manifest works exist
+	listOpts := &client.ListOptions{Namespace: managedClusterName}
+	manifestWorks := &workv1.ManifestWorkList{}
+	if err := r.client.List(ctx, manifestWorks, listOpts); err != nil {
```

Comment:
do you think we should have a labelSelector here?

Reply:
yeah, we can have a label selector